samples/dashboard/demos/climate/demo.js

```diff
@@ -4,66 +4,63 @@ const color1 = Highcharts.Color.parse('#39F');
 const color2 = Highcharts.Color.parse('#6C0');
```

Comment:
Can we use a more meaningful name for variables? F.i. `color_black` or

Reply:
I do not think that this is wise for gradient colors. The value might change and then you have to replace the code lines. Something like `colorMin`, `colorMid`, `colorMax` might be an alternative.

Reply:
It seems that it does not matter soon, because `styledMode` will make the whole concept obsolete.